### ISSUE TITLE
Bug: can't import vite-plugin-wasm 3.4.0 in vite.config.ts

### ISSUE THREAD

**Issue by steel97** [Menci/vite-plugin-wasm#65]
Hello!
I just upgraded to **vite-plugin-wasm 3.4.0** and got this error:
```
failed to load config from D:\Develop\tests\vite-project\vite.config.ts
error when starting dev server:
file:///D:/Develop/tests/vite-project/node_modules/vite-plugin-wasm/exports/import.mjs:1
import { default as m } from "../dist/index.js";
         ^^^^^^^
SyntaxError: The requested module '../dist/index.js' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:134:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:217:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadConfigFromBundledFile (file:///D:/Develop/tests/vite-project/node_modules/vite/dist/node/chunks/dep-BZMjGe_U.js:53862:15)
    at async loadConfigFromFile (file:///D:/Develop/tests/vite-project/node_modules/vite/dist/node/chunks/dep-BZMjGe_U.js:53691:24)
    at async resolveConfig (file:///D:/Develop/tests/vite-project/node_modules/vite/dist/node/chunks/dep-BZMjGe_U.js:53214:24)
    at async _createServer (file:///D:/Develop/tests/vite-project/node_modules/vite/dist/node/chunks/dep-BZMjGe_U.js:43529:18)
    at async CAC.<anonymous> (file:///D:/Develop/tests/vite-project/node_modules/vite/dist/node/cli.js:745:20)
```

**Steps to reproduce**
- create new vite project (vue, typescript)
- install vite-plugin-wasm dependency
- import vite-plugin-wasm in vite.config.ts
- run either `npm run dev` or `npm run build`

My `vite.config.ts`:
```
import { defineConfig } from 'vite'
import vue from '@vitejs/plugin-vue'
import wasm from 'vite-plugin-wasm'

export default defineConfig({
  plugins: [vue(), wasm()],
})

```

NodeJS version: **v20.14.0**
OS: **Windows 11**

**Comment by shubhexists** [comment]
same issue

**Comment by Menci** [comment]
I accidentally committed `"type": "module"` to package.json. The fix #66 is coming.

**Comment by Menci** [comment]
Fixed in v3.4.1
